prjxray/overlay.py

```diff
@@ -0,0 +1,60 @@
+class Overlay(object):
```

Comment:
Tests?

Reply:
Not sure what a proper test for this in prjxray would be.  Any suggestions?

Reply:
Provide a grid and a region dict and ensure that things work as expected?

Reply:
Ok makes sense.  Should that be a new minitests subdirectory?

Reply:
We have https://github.com/SymbiFlow/prjxray/tree/master/tests and I believe we are running them via pytest.

Reply:
https://github.com/SymbiFlow/prjxray/blob/master/Makefile#L59-L60